main.rb

```diff
@@ -0,0 +1,56 @@
+require_relative 'app'
+
+class Interface
+  def initialize
+    @app = App.new
+    display_menu
+  end
+
+  def display_menu
+    loop do
+      print_menu
+      option = gets.chomp.to_i
+      break if option == 7
+
+      handle_option(option)
+    end
+  end
+
+  def print_menu
+    menu_options = "
+1 - List all books
+2 - List all people
+3 - Create a person
+4 - Create a book
+5 - Create a rental
+6 - List all rentals for a given person id
+7 - Exit
+"
+    puts '----Welcome to School Library App!----', '', 'Please choose an option by entering a number:', menu_options
```

Comment:
- [ ] [optional] @JavierAybar It would be nice if this message is shown only when the user starts the app, currently it shows up whenever an option is selected
![image](https://github.com/JavierAybar/OOP-school-library/assets/61408860/74c2c4d9-364c-406d-81a4-f094afd74a72)